README.md

```diff
@@ -1,6 +1,13 @@
 <div align="center">
   <img src="https://github.com/tiny-dnn/tiny-dnn/blob/master/docs/logo/TinyDNN-logo-letters-alpha-version.png"><br><br>
 </div>
+
+-----------------
+
+[![Maintainers Wanted](https://img.shields.io/badge/maintainers-wanted-red.svg)](https://github.com/pickhardt/maintainers-wanted)
+
+## The project may be abandoned since the maintainer(s) are just looking to move on. In the case anyone is interested in continuing the project, let us know so that we can discuss next steps.
```

Comment:
Where? In a new ticket? What are the active communication channel?

Reply:
@bhack right, I will open a ticket in order to point there